index.html

```diff
@@ -10487,7 +10487,16 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-actions">
 			<pdef>aria-actions</pdef>
 			<div class="property-description">
-				<p>Placeholder: text description of aria-actions</p>
+				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger behaviors or operations relevant to the current element.</p>
+				<p>The <code>aria-actions</code> attribute allows an element to reference other interactive elements which trigger actions on the current element. For example, in a web mail application, the currently selected email message may reference action elements such as the <em>message-specific</em> Reply, Forward, and Delete buttons. Actions are triggered by mainstream pointer events (click, tap) or by simulated pointer events from assistive technology.</p>
+				<ul>
+					<li>Authors MUST ensure that related action elements have a label.</li>
+					<li>Authors MUST ensure that related action elements accept a <code>click</code> event handler or default activation behavior.</li>
+					<li>Authors MUST ensure that related action events are not limited to a modality-specific activation such as the <code>keyup</code> or <code>touchend</code> event handlers.</li>
```

Comment:
```suggestion
					<li>Authors MUST ensure that related action elements handle a <code>click</code> event and are not limited to modality-specific activation such as the <code>keyup</code> or <code>touchend</code> events.</li>
```
What do you think about condensing the previous two list items into the above?

Reply:
I have a mild preference for keeping them separate (from a validator implementation perspective), but if you feel strongly about combining them, I'm fine with that.

Reply:
@smhigley I won't change this without another comment from you requesting it.

Reply:
I don't have a strong opinion on this, I'm fine keeping it as-is 👍

Reply:
thanks. marking as resolved.

Reply:
I've come back around on this and like @smhigley's suggestion better. Committing then resolving again.